### PR TITLE
Analytics old advice pages and scripts clean up

### PR DESCRIPTION
**Pull request by ldodds**
Updates to the application following the changes in this PR to clean up the analytics: https://github.com/Energy-Sparks/energy-sparks_analytics/pull/600.

* Bump analytics version, the version with all of the old code removed is is now tagged as 4.0.0
* Remove references to old dashboard config in the application
* Remove some older, unused routes
* Remove `:replace_analysis_pages` feature flag and related code
* Remove `:replace_find_out_mores` feature flag and related code for managing that content

### CODE REVIEW

File: app/services/schools/generate_analysis_chart_configuration.rb [hunk 1]
```diff
@@ -6,7 +6,7 @@ def initialize(
         school,
         aggregated_meter_collection,
         fuel_configuration,
-        dashboard_fuel_types = DashboardConfiguration::DASHBOARD_FUEL_TYPES,
+        dashboard_fuel_types = {},
```
Comment (urbanwide) [inline app/services/schools/generate_analysis_chart_configuration.rb:9]: Not entirely sure what this does but it looks like quite a few things depend on configuration returned later on? Was it intended?

**Review by urbanwide — COMMENTED** [review]
A couple of comments @ldodds - could be more to do with me not understanding how it all fits together though but thought I would mention just in case!

File: spec/controllers/schools/find_out_more_controller_spec.rb [hunk 2]
```diff
@@ -22,67 +22,49 @@
...
     Alerts::GenerateContent.new(school).perform
   end
 
-  around do |example|
-    ClimateControl.modify FEATURE_FLAG_REPLACE_FIND_OUT_MORES: feature_flag do
-      example.run
-    end
-  end
-
   context '#redirect_if_disabled_and_not_admin' do
```
[the remark this answers is not in the document]
Comment (ldodds) [reply spec/controllers/schools/find_out_more_controller_spec.rb:29]: No it needs updating, I'll do that.

**Review by urbanwide — APPROVED** [review]
Looks good @ldodds 👍

**Pull request merged**